frontend/public/components/utils/resource-link.tsx

```diff
@@ -99,11 +100,16 @@ export const ResourceLink = connectToModel(
             title={title}
             className="co-resource-item__resource-name"
             data-test-id={value}
+            data-test={dataTest}
           >
             {value}
           </Link>
         ) : (
-          <span className="co-resource-item__resource-name" data-test-id={value}>
+          <span
+            className="co-resource-item__resource-name"
+            data-test-id={value}
+            data-test={dataTest}
```

Comment:
I've added a `data-test` attribute here so that the tests that depend on `data-test-id` don't fail.